### ISSUE TITLE
command line options rejected

### ISSUE THREAD

**Issue by srittau** [raelgc/jasmine-xml-reporter#2]
Unfortunately, this module does not seem to work anymore with newer Jasmine versions, since its command line options get rejected:

```
srittau@moby:~$ ./node_modules/jasmine-xml-reporter/bin/jasmine.js --junitreport 
Unknown options: --junitreport

[...]
```

This is probably related to commit jasmine/jasmine-npm@88b838b69863abcd07d83221d80a48027648af15, which prevents unknown options in jasmine-npm. (Cf. jasmine/jasmine-npm#53)

**Comment by raelgc** [comment]
@srittau thanks for reporting this and thanks for investigate the reason.

Let me try some experiments and I'll report in few hours.

**Comment by raelgc** [comment]
@srittau Can you update your install and try again?

**Comment by srittau** [comment]
@raelgc Wow, what a fast fix! I've upgraded jasmine-xml-reporter to 1.1.0 and (re-upgraded) jasmine to 2.7.0 and it works flawlessly now.

**Comment by raelgc** [comment]
@srittau Thank you for the nice issue reporting! :)
